meta-oe/recipes-devtools/guider/guider_3.9.7.bb

```diff
@@ -6,21 +6,27 @@ AUTHOR = "Peace Lee <ipeace5@gmail.com>"
 LICENSE = "GPLv2+"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=2c1c00f9d3ed9e24fa69b932b7e7aff2"
 
-PV = "3.9.6+git${SRCPV}"
+PV = "3.9.7+git${SRCPV}"
 PR = "r0"
 
 SRC_URI = "git://github.com/iipeace/${BPN}"
 #SRCREV = "${AUTOREV}"
-SRCREV = "fef25c41efb9bde0614ea477d0b90bd9565ae0b4"
+SRCREV = "948f9fbccddf0225ae4da32f5828d64c4319c8cf"
 
 S = "${WORKDIR}/git"
 R = "${RECIPE_SYSROOT}"
 
-inherit ${@bb.utils.contains("BBFILE_COLLECTIONS", "meta-python2", "distutils", "", d)}
+#inherit distutils
+inherit ${@bb.utils.contains("BBFILE_COLLECTIONS", "meta-python3", "distutils", "", d)}
```

Comment:
this should be `meta-python` not `meta-python3` see https://github.com/openembedded/meta-openembedded/blob/master/meta-python/conf/layer.conf#L7

Reply:
fixed